src/services/navigationBar.ts

```diff
@@ -176,23 +190,36 @@ namespace ts.NavigationBar {
             }
         }
 
+        function hasNonAnonymousFunctionDeclarations(nodes: NodeArray<any>) {
```

Comment:
make this take a `FunctionLikeDeclaration` and call it `functionBodyHasDeclaration`.